### ISSUE TITLE
Arduino to Light Strip signal problem

### ISSUE THREAD

**Issue by phillashcroft** [scottlawsonbc/audio-reactive-led-strip#141]
I have the Wiring done as stated [here](https://learn.adafruit.com/neopixels-on-raspberry-pi/wiring) using this [ESP8266](https://www.amazon.com/gp/product/B01IK9GEQG/ref=oh_aui_detailpage_o02_s00?ie=UTF8&psc=1) this [Light Strip](https://www.amazon.com/gp/product/B00ZHB9M6A/ref=oh_aui_detailpage_o05_s00?ie=UTF8&psc=1) and a [74AHCT125](https://www.amazon.com/gp/product/B00XW2L39K/ref=oh_aui_detailpage_o03_s00?ie=UTF8&psc=1). 

When I ground the 1OE pin, the lights stop. 
When I unplug the 1OE pin, they go kinda to the music but more random than anything. 

Plugging directly from RX pin to DIN on the Strip does nothing as well.  

The arduino itself seems to be functioning properly, but the signal isn't right. 

Please don't mind the poor wiring, it will be all soldered once I see it work.  

![image](https://user-images.githubusercontent.com/27460179/38343062-b2813f2a-3836-11e8-87f6-7b4114f41b28.png)
![image](https://user-images.githubusercontent.com/27460179/38343076-bb252434-3836-11e8-85d6-68220cad697c.png)

**Comment by joeybab3** [comment]
I don't think you need the quad level shifter, have you tried running without it?
